spinnaker-monitoring-daemon/spinnaker-monitoring-daemon.gradle

```diff
@@ -15,7 +17,8 @@
  */
 
 apply plugin: 'nebula.ospackage'
-
+project.tasks.register("publish")
+apply plugin: BintrayPublishPlugin
```

Comment:
Do you know why we're stilling applying this even though I think Cam switched us to publish using the generic maven publishing plugin? Also, why we list the `io.spinnaker.project` plugin in `plugins`, but then never actually apply it? I was assuming it's because the Deck PR wasn't done yet...

Reply:
I think this is correct for these projects...

The io.spinnaker.project is pretty opinionated about a spinnaker jvm based service, where here we only really need the deb publishing to bintray and all the associated config flags that go along with it.

Reply:
So can we just remove the spinnaker gradle plugin entirely? Right now it's listed under `plugins` but with `apply false`, so is it actually still doing anything, or is it just a appendix-like pointless vestige of an earlier time?

Reply:
we want BintrayPublishPlugin that comes from the io.spinnaker.project, and transitively the nebula.ospackage dependency also comes from there

apply false still brings it into the gradle classpath, it just doesn't automatically apply it in the current project

better would maybe be to register BintrayPublishPlugin as an id in the plugin portal (adding an entry for it [here](https://github.com/spinnaker/spinnaker-gradle-project/blob/master/spinnaker-project-plugin/build.gradle#L30))

Reply:
this can become apply plugin: "io.spinnaker.bintray-publish"

do we need to publish the spinnaker-monitoring-thirdparty deb?